lib/member_div.rb

```diff
@@ -1,13 +1,18 @@
 # frozen_string_literal: true
 require 'scraped'
+require 'execjs'
```

Comment:
if you're adding this here, you should really remove it from `scraper.rb` as well (I'll push a commit to do that)